Menus/PinnedToolMenu/Scripts/HintLine.cs

```diff
@@ -0,0 +1,94 @@
+﻿#if UNITY_EDITOR
+using System.Collections;
+using UnityEditor.Experimental.EditorVR.Extensions;
+using UnityEditor.Experimental.EditorVR.Utilities;
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR.Menus
+{
+	public class HintLine : MonoBehaviour
+	{
+		const string k_ShaderLineRadiusPropertyName = "_lineRadius";
+
+		[SerializeField]
+		VRLineRenderer m_ScrollLineRenderer;
+
+		[SerializeField]
+		Color m_VisibleColor = Color.white;
+
+		[SerializeField]
+		Color m_PulseColor = Color.white;
+
+		[SerializeField]
+		MeshRenderer m_MeshRenderer;
+
+		Coroutine m_ScrollArrowPulseCoroutine;
+		float m_PulseDuration;
+		Material m_HintLineMaterial;
+
+		/// <summary>
+		/// Set the width of the line visuals
+		/// </summary>
+		public float LineWidth { set { m_ScrollLineRenderer.SetWidth(value, value); } }
+
+		/// <summary>
+		/// set the start & end positions for the line visuals
+		/// </summary>
+		public Vector3[] Positions { set { m_ScrollLineRenderer.SetPositions(value) ; } }
+
+		void Awake()
+		{
+			m_ScrollLineRenderer.SetVertexCount(4);
+			m_ScrollLineRenderer.useWorldSpace = true;
+			m_ScrollLineRenderer.SetWidth(0f, 0f);
+			m_HintLineMaterial = MaterialUtils.GetMaterialClone(m_MeshRenderer);
+		}
+
+		void OnDestroy()
+		{
+			ObjectUtils.Destroy(m_HintLineMaterial);
+		}
+
+		/// <summary>
+		/// Perform an animated visual pulsing of color
+		/// </summary>
+		public void PulseColor()
+		{
+			if (Mathf.Approximately(m_PulseDuration, 0f) || m_PulseDuration > 0.85f)
+				this.RestartCoroutine(ref m_ScrollArrowPulseCoroutine, AnimatePulseColor());
+		}
+
+		IEnumerator AnimatePulseColor()
+		{
+			const float kTargetDuration = 1f;
+			m_PulseDuration = 0f;
+			var maxShaderLineRadius = new Vector3 (0.03f, 0f, 100f);
+			var minShaderLineRadius = new Vector3 (0.005f, 0f, 100f);
+			var currentVector3ShaderLineRadius = m_HintLineMaterial.GetVector(k_ShaderLineRadiusPropertyName);
+			var currentColor = m_ScrollLineRenderer.colorStart; // The line stand & end colors are the same; fetch only one of them
+			while (m_PulseDuration < kTargetDuration)
+			{
+				var shapedDuration = MathUtilsExt.SmoothInOutLerpFloat(m_PulseDuration / kTargetDuration);
+				var newColor = Color.Lerp(currentColor, m_PulseColor, shapedDuration);
+				m_ScrollLineRenderer.SetColors(newColor, newColor);
```

Comment:
I recommend setting the material color over the line color, but you only have a few points in the line so it is not critical